src/gluonts/ev/batch_aggregations.py

```diff
@@ -0,0 +1,79 @@
+# Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License").
+# You may not use this file except in compliance with the License.
+# A copy of the License is located at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# or in the "license" file accompanying this file. This file is distributed
+# on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+# express or implied. See the License for the specific language governing
+# permissions and limitations under the License.
+
+from dataclasses import dataclass, field
+from typing import Optional
+import numpy as np
+
+from gluonts.ev.helpers import axis_is_zero_or_none
```

Comment:
Generally, it's easiest to use relative import here.

So


```suggestion
from .helpers import axis_is_zero_or_none
```